api/src/main/java/org/apache/iceberg/actions/MigrateTable.java

```diff
@@ -41,6 +41,14 @@ public interface MigrateTable extends Action<MigrateTable, MigrateTable.Result>
    */
   MigrateTable tableProperty(String name, String value);
 
+  /**
+   * Sets the dropBackup property to drop backup table after a successful table migration.
```

Comment:
Sentence is a bit redudant, you can see DeleteOrphanFiles or ExpireSnapshots for more direct examples.  

The previous methods says 'sets a table property' because that's what it does, but here there's no point to say it sets a property.  How about:  'Drops the backup of the original table after a successful migration'.

Reply:
Done, Fixed in latest commit.